Packs/Traceable/Integrations/Traceable/Traceable.py

```diff
@@ -701,20 +841,19 @@ def get_threat_events(
                 self.__query_api_attributes
                 and "isExternal" in self.optional_api_attributes
                 and "apiId" in domain_event
-                and "value" in domain_event["apiId"]
-                and domain_event["apiId"]["value"] is not None
-                and domain_event["apiId"]["value"] != "null"
-                and domain_event["apiId"]["value"] in api_endpoint_details_map
-                and api_endpoint_details_map[domain_event["apiId"]["value"]] is not None
-                and "isExternal" in api_endpoint_details_map[domain_event["apiId"]["value"]]
-                and api_endpoint_details_map[domain_event["apiId"]["value"]][
+                and domain_event["apiId"] is not None
+                and domain_event["apiId"] != "null"
+                and domain_event["apiId"] in api_endpoint_details_map
+                and api_endpoint_details_map[domain_event["apiId"]] is not None
+                and "isExternal" in api_endpoint_details_map[domain_event["apiId"]]
+                and api_endpoint_details_map[domain_event["apiId"]][
```

Comment:
Please use the dict `get` method
https://www.w3schools.com/python/ref_dictionary_get.asp